python/ray/data/_internal/logical/rules/operator_fusion.py

```diff
@@ -108,10 +115,13 @@ def _can_fuse(self, down_op: PhysicalOperator, up_op: PhysicalOperator) -> bool:
 
         # We currently only support fusing for the following cases:
         # - MapOperator -> MapOperator
-        # - MapOperator -> AllToAllOperator
+        # - TaskPoolMapOperator -> AllToAllOperator
```

Comment:
why isn't actorpoolmapoperator -> alltoallloperator not supported? add a comment about this?

Reply:
Correct me if I'm wrong. For `map -> AllToAll`, we are going to fuse them as one AllToAll. But AllToAll doesn't support running with actors. Will add a comment.

Reply:
Got it, so this was a bug before